h/services/user_delete.py

```diff
@@ -338,13 +362,13 @@ def update(self, model_class, select_stmnt, values: dict) -> list:
             .values(values)
             .returning(model_class.id)
         )
+
         if updated_ids:
-            log.info(
-                "Updated %d rows from %s", len(updated_ids), model_class.__tablename__
-            )
+            log.info(f"{log_message}: %s", ", ".join(str(id_) for id_ in updated_ids))
```

Comment:
_If_ anything was actually updated then it logs the message given by the caller along with the id's of the updated rows.